.github/workflows/release-gh-pages.yml

```diff
@@ -1,26 +1,14 @@
-name: GitHub Pages
+name: Build and release site to GitHub Pages
 
 on:
-  pull_request:
-    branches: [main]
   push:
     branches: [main]
+  schedule:
+    # daily build
+    - cron:  '15 8 * * *'
```

Comment:
Unfortunately GitHub Actions do not support the non-standard `daily` option :(
Minor but can we change this to `'0 0 * * *'` to be more intuitive?

Reply:
Ok, changed!

I was using a more "peculiar" cron expression because of [this](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events):

> The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.

But it's not important for our use case.